### ISSUE TITLE
Cannot activate Burn after reading

### ISSUE THREAD

**Issue by KooBonchan** [halogenandtoast/ArkhamHorror#1288]
I cannot activate Burn After Reading after 2nd activation in full scenario.
Since Burn after Reading has (Limit twice per game), Its activation should be limited twice per scenario, not per campaign.

Reference text: Father Mateo Investigator ability (Limit once per game)

file: https://arkham-horror-bugs.s3.amazonaws.com/exports/0e47257d63c3ede1aaaf5df27232add885ec5736f7ddd13b80c62ceb29ef3cb8.json

**Comment by halogenandtoast** [comment]
Yeah the issue was the uses just not being cleared out between campaigns. Fix incoming, but it won't do anything in this scenario. However you will be able to use it again in the next one.
